docs/code/MlNetCookBook.md

```diff
@@ -1022,3 +1023,46 @@ ITransformer loadedModel;
 using (var fs = File.OpenRead(modelPath))
     loadedModel = newContext.Model.Load(fs);
 ```
+
+## How can I read and write binary data?
```

Comment:
Cookbook examples all have corresponding tests in `Tests/Scenarios/Api/CookbookSamples/`. Could you please add one for this too?

Reply:
I must be missing something when I try to add the example. It's not finding the `CreateDataView` method on the ML context. It may be missing a reference for it, but I'm not sure which one it needs.

Reply:
@rogancarr I switched to `LoadFromEnumerable` which is what the sample is already using. Hopefully the test looks good now.